ex1/index.html

```diff
@@ -16,6 +16,14 @@ <h2>This is my blue button</h2>
     id="demo">
     Hit Me!!!
   </button>
+  <img src="noajz.jpg" alt="Smiley face" height="300" width="300" id="noaIMG">
```

Comment:
change to noa-img and not noaIMG

Reply:
also it's not an id, it should be in a class

Reply:
i need the id for the function changeColor kaay